src/main/java/com/example/bookstore/model/User.java

```diff
@@ -50,6 +53,10 @@ public class User implements UserDetails {
             inverseJoinColumns = @JoinColumn(name = "role_id"))
     private Set<Role> roles;
 
+    @OneToOne(mappedBy = "user", fetch = FetchType.LAZY,
+            cascade = CascadeType.ALL)
```

Comment:
does it make sense to add `orphanRemoval = true` since shopping cart can't exist without user?